fix: update README (#12)
FromDoppler/doppler-swarm: README.md

Here we will store the configuration of our experimental Docker Swarm Architecture.

> It also define the image for _Doppler Swarm Proxy_ with the required configuration.

Reviewer: ```suggestion
It also defines the image for _Doppler Swarm Proxy_ with the required configuration.
```

> **IMPORTANT**: the folder _swarm-stack_ is very useful to test local, but its contents could be non-updated.

Reviewer 2: So, all modifications we need in doppler stack now are to be deployed in the other repository, not just sensitive data right?

Author: Right, the scope of the [another repository (doppler-swarm-devops)](https://github.com/MakingSense/doppler-swarm-devops) is allowing us continuous deployment of our infrastructure (services, instances, certificates, production sensitive configuration data, etc).

The scope of the current repository is restricted to define our reverse proxy image.

The problem is how to test the complete environment for people without access to our _doppler-swarm-devops_ repository. Also, how to deal with QA environments.

Further work could be moving Nginx configuration to the _doppler-swarm-devops_ repository and dispense with this repository and the proxy image, but it does not solve the issue mentioned previously.

Reviewer 2: Right so then I should update both for now. Mostly the other repository should be up with the latest changes. Thanks @andresmoschini

> **IMPORTANT**: the folder _swarm-stack_ is very useful to test local, but its contents could be non-updated.

Reviewer: ```suggestion
**IMPORTANT**: the folder _swarm-stack_ is very useful to test locally, but its contents could be not up-to-date.
```

> See the production content in the repo [oppler-swarm-devops](https://github.com/MakingSense/doppler-swarm-devops).

Reviewer 3: ```suggestion
See the production content in the repo [doppler-swarm-devops](https://github.com/MakingSense/doppler-swarm-devops).
```
